functions/Get-DbaDatabase.ps1

```diff
@@ -240,16 +293,22 @@
 				foreach ($db in $inputobject) {
 					
 					$Notes = $null
+					
 					if ($NoFullBackup -or $NoFullBackupSince) {
 						if (@($db.EnumBackupSets()).count -eq @($db.EnumBackupSets() | Where-Object { $_.IsCopyOnly }).count -and (@($db.EnumBackupSets()).count -gt 0)) {
 							$Notes = "Only CopyOnly backups"
 						}
 					}
-					Add-Member -Force -InputObject $db -MemberType NoteProperty BackupStatus -value $Notes
+					
+					$lastusedinfo = $dblastused | Where-Object { $_.dbname -eq $db.name }
 					
 					Add-Member -Force -InputObject $db -MemberType NoteProperty -Name ComputerName -value $server.NetName
 					Add-Member -Force -InputObject $db -MemberType NoteProperty -Name InstanceName -value $server.ServiceName
 					Add-Member -Force -InputObject $db -MemberType NoteProperty -Name SqlInstance -value $server.DomainInstanceName
+					Add-Member -Force -InputObject $db -MemberType NoteProperty -Name LastRead -value $lastusedinfo.last_read
+					Add-Member -Force -InputObject $db -MemberType NoteProperty -Name LastWrite -value $lastusedinfo.last_write
```

Comment:
This is not 'unhiding' a column, this is additional functionality from multiple sources.